app/views/article/record/_html_fulltext.erb

```diff
@@ -0,0 +1,13 @@
+<% doc_presenter = presenter(document) -%>
+<div class="section-container-heading" data-toggle="collapse" href="#toggleFulltext" aria-expanded="false" aria-controls="toggleFulltext">
+  <h2>Full text</h2>
+</div>
```

Comment:
I haven't checked this branch out yet (in the middle of a bunch of changes in my working branch at the moment), but a couple questions I have:

Is this `div` keyboard accessible (e.g. can you tab to it and hit enter to open/close)?  Does the cursor indicate that the heading element is clickable (it didn't look like it from the gif, but can be hard to tell).